apps/howoldami/howoldami.star

```diff
@@ -0,0 +1,147 @@
+        delay = 1000,
+        child = render.Box(
+            render.Column(
+                main_align = "start",
+                expanded = True,
+                children = [
+                    render.Row(cross_align = "center", expanded = True, main_align = "space_around", children = [render.Text("How Old Am I?")]),
+                    render.Padding(
+                        child = render.Row(
+                            main_align = "space_between",
+                            expanded = True,
+                            children = [
+                                render.Text("Secs:", font = font, color = "#ff0000"),
+                                render.Animation(
+                                    children = secondList,
+                                ),
+                            ],
+                        ),
+                        pad = (0, 0, 0, 1),
+                    ),
+                    render.Padding(
+                        child = render.Row(
+                            main_align = "space_between",
+                            expanded = True,
+                            children = [
+                                render.Text("Mins:", font = font, color = "#f2c57c"),
+                                render.Animation(
+                                    children = minuteList,
+                                ),
+                            ],
+                        ),
+                        pad = (0, 0, 0, 1),
+                    ),
+                    render.Padding(
+                        child = render.Row(
+                            main_align = "space_between",
+                            expanded = True,
+                            children = [
+                                render.Text("Hours:", font = font, color = "#55ef67"),
+                                render.Animation(
+                                    children = hourList,
+                                ),
+                            ],
+                        ),
+                        pad = (0, 0, 0, 1),
+                    ),
+                    render.Padding(
+                        child = render.Row(
+                            main_align = "space_between",
+                            expanded = True,
+                            children = [
+                                render.Text("Days:", font = font, color = "#00a9aa"),
+                                render.Text("%d" % float(days), font = font, color = "#00a9aa"),
+                            ],
+                        ),
+                        pad = (0, 0, 0, 1),
+                    ),
+                ],
+            ),
+        ),
+    )
+
+def get_schema():
+    options = [
+        schema.Option(display = "US/Central", value = "US/Central"),
+        schema.Option(display = "US/Eastern", value = "US/Eastern"),
+    ]
+
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
```

Comment:
Have you seen the [`Datetime`](https://github.com/tidbyt/pixlet/blob/main/docs/schema/schema.md#datetime) schema field? Should be a great fit for this data and make it a lot easier for users to enter correct information.

Reply:
I thiiiiink it'll also give you the time adjusted to UTC, removing the need for juggling timezones.

Reply:
Good to know! Very useful here